### DISCUSSION TITLE
Official documentation should be updated

### DISCUSSION THREAD

**Discussion by dcb9** [spatie/laravel-open-telemetry#21] [Q&A]
https://spatie.be/docs/laravel-open-telemetry/v1/installation-setup#content-setting-up-jaeger-via-docker-locally

**Comment by freekmurze** [comment]
Feel free to send a PR with the changes you'd like to see.

**Reply by dcb9** [reply]
Hi @freekmurze, what I mean actually is that the official website should sync github repo /docs directory.  Here are two docs which are not synchronized: 

https://github.com/spatie/laravel-open-telemetry/blob/83ed786386d86b87a5bc55e383b38c6663f92153/docs/installation-setup.md#setting-up-zipkin-via-docker-locally

https://spatie.be/docs/laravel-open-telemetry/v1/installation-setup#content-setting-up-jaeger-via-docker-locally
